integrationTests/package.json

```diff
@@ -0,0 +1,143 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "private": true,
+  "scripts": {
+    "depcheck": "depcheck",
+    "lint": "eslint . --ext .ts,.js --quiet --fix",
+    "pretest": "npm run lint && npm run depcheck",
+    "test:ts": "nyc mocha \"tests/*.test.ts\"",
+    "test:js": "nyc mocha \"tests/*.test.js\"",
+    "test": "npm run test:ts && npm run test:js",
+    "redis": "npm run redis --prefix ../packages/core",
+    "redis:shutdown": "npm run redis:shutdown --prefix ../packages/core",
+    "redis:debug": "npm run redis:debug --prefix ../packages/core",
+    "redis:debug:shutdown": "npm run redis:debug:shutdown --prefix ../packages/core",
+    "bundle:core": "npm pack ../packages/core && mv commerce-apps-core*.tgz commerce-apps-core.tgz",
+    "bundle:sdk": "npm pack ../packages/generator && mv commerce-sdk-*.tgz commerce-sdk.tgz",
+    "bundle:all": "npm run bundle:core && npm run bundle:sdk",
+    "bundle:install": "npm run bundle:all && npm install",
+    "bundle:clean:install": "npm run clean:all && npm install",
```

Comment:
I think this script will fail every time. Can u please test this bundle:clean:install? if it is not used anywhere else pls remove it..

Reply:
Which bit do you think will fail? I saw an issue with the rm part of the clean failing when there was no file, I've added a fix for that.